common/djangoapps/enrollment/api.py

```diff
@@ -132,6 +132,38 @@ def get_enrollment(user_id, course_id):
     return _data_api().get_course_enrollment(user_id, course_id)
 
 
+def get_roster(course_id):
+    """Retrieves username, email, name, and global anonymous_user_id
```

Comment:
This message should be more generalized. As is, this is a maintenance burden; you need to edit the docstring anytime you update the return fields; which is to say, the documentation will probably fall out of sync.

I don't think the comment needs to explicitly list every key in the dictionary, we can just say "Retrieve a list of users enrolled in a course"